projects/fmcjesdadc1/common/fmcjesdadc1_bd.tcl

```diff
@@ -1,11 +1,22 @@
+#
+# Parameter description:
+#   JESD_MODE : Used link layer encoder mode
+#      64B66B - 64b66b link layer defined in JESD 204C
+#      8B10B  - 8b10b link layer defined in JESD 204B
+#
+#   RX_JESD_M : Number of converters per link
+#   RX_JESD_L : Number of lanes per link
+#   RX_JESD_S : Number of samples per frame
+#   RX_JESD_NP : Number of bits per sample
+#
```

Comment:
JESD_MODE is not useful for this project as the device doesn't support JESD204C.

Reply:
The number of lanes should influence the buswidth of the source of the DMA

Reply:
The number of converters (M) should influence the CPACK connections

Reply:
Modified according to the above suggestions.